experimenter/experimenter/experiments/tests/test_models.py

```diff
@@ -1893,8 +1885,7 @@ def test_audience_url(self):
         self.assertEqual(
             experiment.audience_url,
             (
-                "/nimbus/?application=firefox-desktop&channel=release&firefox_min_version=100.%21"
-                f"&feature_configs={feature1.id}&feature_configs={feature2.id}"
+                "/nimbus/?application=firefox-desktop&channel=release"
```

Comment:
I guess dict insertion order = iteration order makes this a stable URL?

Reply:
Oh it's not a dict, it's a list of tuples, so you can have keys repeated.